plugins/module_utils/data_set.py

```diff
@@ -1025,8 +1025,10 @@ def create(
         except exceptions.ZOAUException as create_exception:
             raise DatasetCreateError(
                 name,
-                create_exception.response.rc,
-                create_exception.response.stdout_response + create_exception.response.stderr_response
+                # create_exception.response.rc,
```

Comment:
Hey Rich, we can remove this instead of commenting them, those have been recently added so no problem, but before doing that. I'm curious about removing the response access, Did you find out that the structure at ZOAU docs https://www.ibm.com/docs/en/zoau/1.3.x?topic=apis-exceptions, does not align with the python ZOAUException structure ? Meaning, ZOAUException is not having a response attribute?

Reply:
I'll remove them... yes, the zoau exception returned does not contain 'response' as an element, which crashes things.